src/modules/Convolution/Convolution.js

```diff
@@ -0,0 +1,67 @@
+module.exports = exports = function(pixels, constantFactor, kernelValues){
+	let kernel = kernelGenerator(constantFactor, kernelValues), oldpix = pixels;
```

Comment:
Aha! @Mridul97 Don't you think this will just copy the pixels object by reference? Shouldn't we use lodash here?